geth/api/backend_test.go

```diff
@@ -80,6 +82,31 @@ func (s *BackendTestSuite) StopTestBackend() {
 	require.False(s.backend.IsNodeRunning())
 }
 
+// EnsureNodeSync ensures that synchronization of the node is done once and that it
+// is done properly else, the call will fail.
+func (s *BackendTestSuite) EnsureNodeSync() {
```

Comment:
Duplicate with `BaseTestSuite`?

Reply:
BackendTestSuite does not Embed `BaseTestSuite` hence the need to duplicate things.

Reply:
Let's fix it anyway.

Reply:
Alright.